lib/redirect.dart

```diff
@@ -29,7 +29,9 @@ final _matchers = <RegExp, Uri Function(String)>{
   RegExp(r'^/opened/([A-Za-z0-9\-]+)$'): _openedIssues.resolve,
   RegExp(r'^/area/([A-Za-z0-9\-]+)$'): (match) => _listIssues.replace(
         queryParameters: {
-          'label': 'area-$match',
+          'q': [
+            'label:area-$match',
+          ].join(' '),
```

Comment:
This part isn't necessary in this case, but I added it to be consistent and in case any other parameters are added in the future.